javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,41 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  
+    studentName === "" ? console.log("Please add a name!"):undefined;
```

Comment:
The ternary operator is used here but the undefined part is unnecessary. It would be clearer to use an if statement.
And if you try to add an empty name, it will still be added.
So it is best to make a consistent if, else if kind of statements.
```suggestion
if (studentName === "") {
    console.log("Please add a name!");
    return;
  }```